crates/shared/src/http_solver/model.rs

```diff
@@ -500,20 +502,8 @@ mod tests {
         };
         let model = BatchAuctionModel {
             tokens: btreemap! {
-                buy_token => TokenInfoModel {
-                    decimals: Some(6),
-                    alias: Some("CAT".to_string()),
-                    external_price: Some(1.2),
-                    normalize_priority: Some(1),
-                    internal_buffer: Some(U256::from(1337)),
-                },
-                sell_token => TokenInfoModel {
-                    decimals: Some(18),
-                    alias: Some("DOG".to_string()),
-                    external_price: Some(2345.0),
-                    normalize_priority: Some(0),
-                    internal_buffer: Some(U256::from(42)),
-                }
+                buy_token => TokenInfoModel {decimals:Some(6),alias:Some("CAT".to_string()),external_price:Some(1.2),normalize_priority:Some(1),internal_buffer:Some(U256::from(1337)), accepted_for_internalization: true },
+                sell_token => TokenInfoModel {decimals:Some(18),alias:Some("DOG".to_string()),external_price:Some(2345.0),normalize_priority:Some(0),internal_buffer:Some(U256::from(42)), accepted_for_internalization: true }
```

Comment:
`rustfmt` doesn't format this because it's inside a macro. (Yet another reason to avoid macros.)

I still think this formatting should be cleaned up manually by splitting it into multiple lines.

Reply:
Fixed.